render/src/main/java/org/codice/imaging/nitf/render/NitfRenderer.java

```diff
@@ -218,6 +221,44 @@ private void renderJPEG2k(final ImageSegment imageSegment, final Graphics2D targ
             }
         }
 
+        /* If R, G, B are not present, the default displayable band
+        is the LU band
+         */
+        if (imageBands.isEmpty()) {
```

Comment:
M and LU should be separate cases which I've tried to demonstrate in my comments. Hopefully that clears it up.